### ISSUE TITLE
How can I set optimizer group with weight decay?

### ISSUE THREAD

**Issue by sudo-rm-covid19** [open-mmlab/mmdetection#2995]
During finetuning, I would like to reduce the learning rate for backbone layers. Is there a feature that supports this operation?
Thanks.

**Comment by hellock** [comment]
Yes you can build your own optimizer. Reference: https://github.com/open-mmlab/mmdetection/blob/master/mmdet/apis/train.py#L134
You may either:
1. Define a new OptimizerContructor. (recommended, with no need to modify the core library)
2. Define a new optimizer directly. (need to modify the core library)
